pennylane/ops/op_math/controlled.py

```diff
@@ -392,20 +394,24 @@ class Controlled(SymbolicOp):
 
     >>> op.base
     RX(1.234, wires=[1])
+
     >>> op.data
     (1.234,)
+
     >>> op.wires
     Wires([0, 1])
+
     >>> op.control_wires
     Wires([0])
+
     >>> op.target_wires
     Wires([1])
 
     Control values are lists of booleans, indicating whether or not to control on the
     ``0==False`` value or the ``1==True`` wire.
 
     >>> op.control_values
-    [0]
+    [False]
 
     Provided control values are converted to booleans internally, so
     any "truthy" or "falsy" objects work.
```

Comment:
Can we also update the line below? Unfortunately I am not able to comment at the correct place...

Reply:
@JerryChen97  Do you want me to just delete it? I'm actually not quite sure what should belong here, documentation wise.

Reply:
I just commited what I meant @austingmhuang

Reply:
Thanks @JerryChen97 !